src/app/components/shell/header/menus/upper-menu/upper-menu.js

```diff
@@ -9,7 +9,7 @@ function BlogItem() {
         'pages?type=news.newsArticle&fields=id&limit=3'
     );
 
-    const shouldDisplay = data && data.items.length > 0;
+    const shouldDisplay = data && data.items && data.items.length > 0;
 
```

Comment:
BTW, tutor's using optional chaining which makes this: 
`const shouldDisplay = !!data?.items?.length`
https://github.com/openstax/tutor-js/blob/main/babel.config.js#L16

Reply:
Woah: This plugin is included in @babel/preset-env